config/sync/field.field.node.article.field_a_content_author.yml

```diff
@@ -0,0 +1,19 @@
+uuid: bb17fc7d-bbf5-4746-96f2-ffb0d6ba5d2a
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_a_content_author
+    - node.type.article
+id: node.article.field_a_content_author
+field_name: field_a_content_author
+entity_type: node
+bundle: article
+label: 'Content Author'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings: {  }
+field_type: string
```

Comment:
Change this to be a select list like recipe content type and also reuse the `content_author_allowed_values_function`

Reply:
done